basic/html-tag/html-tags.md

```diff
@@ -0,0 +1,176 @@
+
+```html
+<h1>Main Header</h1>
+<h2>Subheading</h2>
+```
+
+### The Paragraph Tag
+
+The `<p>` tag is used to indicate the start of a new paragraph. Be careful not to forget to close off the tag `</p>` to mark the end of that particular paragraph. 
+
+```html
+<body>
+<p>Lorem ipsum dolor paragraph </p>
+</body>
+```
+
+### The Div Tag
+
+The `<div>` tag is used for secluding a portion of the content to apply styling. Different design functions like ‘style’ and ‘padding’ can be implemented in the `<div>` tag.
+
+```html
+<div style="border: 3px solid #000; padding: 20px;">
+  <p>10x Frontend Developer</p>
+</div>
+```
+
+As you can see above, the ‘style’ and ‘padding’ functions are only applied to the paragraph “10x Frontend Developer ’’.
+
+### The Image Tag
+
+The `<img>` tags are used when inserting images into the web page. Images in your device storage can easily be inserted into the web page by using this tag along with the ‘alt’ function to describe the image.
+
+```html
+<img src="image.jpg"
+ alt="A Cat">
+```
+
+### The Hyperlink Tag
+
+The Hyperlink tag is used when inserting links to external resources or websites. When a user clicks on a text embedded in this tag, it automatically redirects them to an external website.
+
+```html
+<a href="https://www.HappyFeet.com"> Available at HappyFeet.com</a>
+```
+
+### The List Tags
+
+As the name implies,  the list tags are used to create lists on the web page. `<ul>` creates an “Unordered List” while `<ol>` creates an “Ordered List”.
+
+```html
+<ul>
+  <li>Micheal Scofield</li>
+  <li>Barry Allen</li>
+  <li>Frank Ocean</li>
+</ul>
+
+<ol>
+  <li>Eggs</li>
+  <li>Bread</li>
+  <li>Milk</li>
+</ol>
+```
+
+### The Iframe Tag
+
+The `<iframe>` tag is used when inserting external files like thumbnails into your web page. This is essential for adding external media to your web page.
+
+```html
+<iframe src="https://www.HappyFeet.com"
+ width="400" height="200"></iframe>
+```
+
+# Implementing HTML Tags
+
+The above tags are important parts of HTML and can not be overlooked. It is important for any web developer to master or at least have an idea of the tags, their uses and how to adequately implement them to create a complete web page. If a tag is not properly implemented, like not being closed off properly, the tag is rendered invalid and hence can not affect the web page.
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+  <h1>Hello World</h1>
+</head>
+
+<body>
+  <p> This is my first webpage. These are my favorite musicians.
+  <ol>
+    <li>Brent Faiyaz</li>
+    <li>Drake</li>
+    <li>Burna Boy</li>
+  </ol>
+  <p>
+  <div style="border: 3px solid #222; padding: 20px;">
+    Check me out at <a href="https;//www.HappyFeet.com">HappyFeet.com</a>
+  </div>
+  </p>
+  </p>
+</body>
+
+</html>
+```
+
+The image below shows the results of the HTML code above:
+
+![Capture.PNG](Capture.PNG)
```

Comment:
Capture.PNG needs to be globally available. You can upload the file at Google drive and use that - 

How to:  https://www.labnol.org/google-drive-image-hosting-220515